template/client/store/actions.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import * as mutationTypes from '~/types/mutations'
+import type {
+  StateType
+} from '~/types'
+import type {
+  ActionContext
+} from 'vuex'
+import comments from '~/logics/api/comments'
+
+export const actions = {
```

Comment:
I would go for `export default` here, because we won't export anything else.
The same applies for mutation/actions/etc.